contracts/scripts/test/measure_gas.ts

```diff
@@ -33,85 +33,102 @@ async function logGasForTransfers() {
     th = new TokenHelper(fx);
     const blsWallets = await th.walletTokenSetup();
 
-    // encode transfer to consecutive addresses of 1*10^-18 of a token
-    // signed by first bls wallet
-    const txs: Bundle[] = [];
-    const startNonce: number = (await blsWallets[0].Nonce()).toNumber();
-    let nonce = startNonce;
+    const nonce: number = (await blsWallets[0].Nonce()).toNumber();
     console.log(
       "airdropper balance before",
       await th.testToken.balanceOf(blsWallets[0].address),
     );
 
-    const AddressZero = "0x" + "0".repeat(40);
-
     console.log("Signing txs from nonce", nonce);
+
+    let actionsArr = [];
+    let encodedFunctions = [];
```

Comment:
These vars are never reassigned and so can be `const`s.
```suggestion
    const actionsArr = [];
    const encodedFunctions = [];
```